index.html

```diff
@@ -0,0 +1,52 @@
+<link rel="stylesheet" href="css/style.css">
+<link href="https://fonts.googleapis.com/css?family=Poppins:200i,300,400&amp;display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <nav id="navbar">
+    
+       <a href="#welcome-section">About</a>
+       <a href="#projects">Work</a>
+       <a href="#conact">Contact</a>
+    
+
+  </nav>
+  
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<div id="welcome-section">
+  <h1>Hey I am Mimic</h1>
+  <h4 id="head4">a web developer</h4>
+</div>
```

Comment:
Semantically speaking, It's incorrect to make `<div>` elements to warp the header part of the page. You can use `<header>` elements. [more](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure)